### ISSUE TITLE
ERROR    SNATCHQUEUE-SNATCH-328361 :: [b34f1df] DelugeD: Failed Sending Torrent

### ISSUE THREAD

**Issue by barukh27** [pymedusa/Medusa#6495]
Hi there,

I'm back here because, once again, I'm getting this issue. The issue got temporally fixed, thanks to medariox, by adding  scgi:// before the localhost. Unfortunately, today my log showed this: 

2019-04-11 22:58:11 ERROR    SNATCHQUEUE-SNATCH-328361 :: [b34f1df] DelugeD: Failed Sending Torrent
Traceback (most recent call last):
  File "/opt/medusa/medusa/clients/torrent/generic.py", line 253, in send_torrent
    r_code = self._add_torrent_uri(result)
  File "/opt/medusa/medusa/clients/torrent/deluged.py", line 66, in _add_torrent_uri
    remote_torrent = self.drpc.add_torrent_magnet(result.url, options, result.hash)
  File "/opt/medusa/medusa/clients/torrent/deluged.py", line 259, in add_torrent_magnet
    self.connect()
  File "/opt/medusa/medusa/clients/torrent/deluged.py", line 191, in connect
    self.client.connect()
  File "/opt/medusa/ext/deluge_client/client.py", line 74, in connect
    self._connect()
  File "/opt/medusa/ext/deluge_client/client.py", line 88, in _connect
    self._socket.connect((self.host, self.port))
  File "/usr/lib/python2.7/ssl.py", line 876, in connect
    self._real_connect(addr, False)
  File "/usr/lib/python2.7/ssl.py", line 863, in _real_connect
    socket.connect(self, addr)
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

What can I do? Any hint?

Thanks in advance.

**Comment by p0psicles** [comment]
In stead of localhost try the ip of the machine

**Comment by barukh27** [comment]
> 
> 
> In stead of localhost try the ip of the machine

Thank you for your comments.

I added the IP and it didn't fix the problem.

**Comment by medariox** [comment]
The machine you are trying to connect to is refusing your request. This is not a Medusa related issue.

**Comment by barukh27** [comment]
> 
> 
> The machine you are trying to connect to is refusing your request. This is not a Medusa related issue.

The confusion comes from the fact that every time I restart medusa it works again even if it is for a short period of time. Any clue?

Thanks

**Comment by barukh27** [comment]
Hi there again!

I made a fresh install and I think that the problem came from some incompatibility issues with Radarr. I uninstalled Radarr and know everything seems to work fine but I'm having issues with the locales, I can't set them manually (I followed the FAQs instructions and tried to set them manually with the dpkg-reconfigure locales command, I rebooted the machine but anything changed).

I'll keep you updated.

Thanks.
